docs/deploy/replication.mdx

```diff
@@ -147,7 +147,8 @@ sudo systemctl start postgresql
 
 ### Verification
 
-Now you can verify the replication. The data inserted on the primary server should appear on the standby servers, confirming that streaming replication with ParadeDB is working correctly.
+Now you can verify the replication. The data inserted on the primary server should appear on the standby servers, confirming that streaming replication with ParadeDB is working correctly. Note
```

Comment:
If the destination table needs to start off empty shouldn't this be said before verification.